### ISSUE TITLE
Not Disabled partner customization like Motorola's 4 columns

### ISSUE THREAD

**Issue by ivanosh** [amirzaidi/Launcher3#214]
Hello. I'm a Moto Z Play user and still have 4 columns in launcher. It's not a bug for me, but you have discrepancy between launcher and its description.
It's a great idea to port Pixel Launcher to other devices. The main thing is to keep it in pure condition and do not add any extra features so as not to become a typical customized launcher.
Thank you for your work!
![photo_2017-09-04_11-22-53](https://user-images.githubusercontent.com/5702943/30020221-6f24aaca-9163-11e7-8eca-a5550fad647b.jpg)

**Comment by abdullahibneat** [comment]
Change the DPI settings for the device. [Here](https://www.google.co.uk/amp/s/forum.xda-developers.com/moto-z-play/how-to/howto-change-dpi-root-t3528908/amp/) is a quick way of doing it through ADB.

**Comment by ivanosh** [comment]
@abdullahibneat no, 4 column is comfortable for me:) I'm just saying that the description of launcher is not actual

**Comment by abdullahibneat** [comment]
Nowhere in the description is stated you'll get 5 columns. Unfortunately, this depends on the device in use, as different devices use different DPI settings.

You may close this issue if you're happy.

**Comment by cyberon** [comment]
This also happens on the Moto G5 Plus, the icons are huge immo.

**Comment by ivanosh** [comment]
@abdullahibneat 
<img width="431" alt="screen shot 2017-09-08 at 20 13 23" src="https://user-images.githubusercontent.com/5702943/30225273-353228c0-94d2-11e7-9613-aff49ba2a763.png">

**Comment by RikKoedoot** [comment]
@ivanosh this is disabled. but if you have the wrong dpi there is not much to fix for the launcher

**Comment by amirzaidi** [comment]
I disabled the forced 4 columns, but if your DPI is too high you'll still see only 4 columns. Lower your DPI to get 5.

**Comment by asianfez123** [comment]
Figured out the issue on my Z Play when trying to find out why my multi window was stuck at 50/50 and could not be adjusted. Change the apparently built-in DPI setting in Android 7.0 Nougat or higher. 

By changing the 'Display Size' option under 'Display' under the 'Settings', from the Motorola default to small, I was now able to slight adjust the size of the multi windows (from only 50/50 to also including ~35/65, which is better than nothing). However, after doing this it also resized everything on my phone smaller, including my app icon sizes, which now fits apps 5 wide on the home screen and app drawer. 

*Note: This made the text a bit too small for my liking so I also increased the 'Font Size' in the same settings location to 'Large' and it feels all good again. ALTERNATIVELY,  change the 'Smallest Width' setting in developer options from the default 360 to a minimum of 380 to get an extra column.
